podaac/subsetter/subset.py

```diff
@@ -1223,6 +1223,8 @@ def subset(file_to_subset, bbox, output_file, variables=None,
 
     if variables:
         variables = [x.replace('/', GROUP_DELIM) for x in variables]
+        if has_groups:
+            variables = [GROUP_DELIM + x if x[0:2] != GROUP_DELIM else x for x in variables]
```

Comment:
```suggestion
            variables = [GROUP_DELIM + x if !x.startswith(GROUP_DELIM) else x for x in variables]
```

Reply:
oops, think I got my python syntax wrong; think it is supposed to be
`if not x.startswith(GROUP_DELIM)`